pkg/loader/loader_test.go

```diff
@@ -5,6 +5,10 @@ import (
 	"encoding/json"
 	"testing"
 
+	"os"
+
+	"github.com/gptscript-ai/gptscript/pkg/types"
+
```

Comment:
nit: move `"os"` up with the other standard library packages and put `"github.com/gptscript-ai/gptscript/pkg/types"` with the other non-standard library packages.

Reply:
no idea, why those were not sorted automatically...